extension/src/components/OpenLoans.tsx

```diff
@@ -1,15 +1,26 @@
-import { Box, Button, FormControl, FormErrorMessage, HStack, Image, VStack } from "@chakra-ui/react";
+import {
+    Box,
+    Button,
+    FormControl,
+    FormErrorMessage,
+    HStack,
+    Image,
+    SkeletonText,
+    useInterval,
+    VStack,
+} from "@chakra-ui/react";
 import { Accordion, AccordionButton, AccordionIcon, AccordionItem, AccordionPanel } from "@chakra-ui/react";
 import Debug from "debug";
 import moment from "moment";
 import * as React from "react";
 import { useAsync } from "react-async";
 import { repayLoan } from "../background-proxy";
+import { getBlockHeight } from "../background-proxy";
 import { LoanDetails } from "../models";
 import Btc from "./bitcoin.svg";
 import Usdt from "./tether.svg";
 
-const error = Debug("openloans:error");
+const debug = Debug("openloans:error");
```

Comment:
I prefer not renaming it. 
`debug` is for printing _normal_ messages
`error` is for printing _error_ messages.

Reply:
I only changed this because the I got `yarn tsc` errors with the `error` variable. I'll give it another try :)